src/seedu/addressbook/AddressBook.java

```diff
@@ -400,7 +400,8 @@ private static String executeAddPerson(String commandArgs) {
         // add the person as specified
         final String[] personToAdd = decodeResult.get();
         addPersonToAddressBook(personToAdd);
-        return getMessageForSuccessfulAddPerson(personToAdd);
+        return String.format(MESSAGE_ADDED,
```

Comment:
BTW, the commit message is misleading, you are not extracting method here from code, you are inlining the code from a method at its place of invocation.

even when we consider this as a good refactoring, I don't see the commit for removing the now redundant method `getMessageForSuccessfulAddPerson()` :-)